### PR TITLE
feature: ECA-9 routing

### PR DESCRIPTION
**Pull request by valkandyba**

### CODE REVIEW

File: src/components/Layout/Header/Header.tsx [hunk 1]
```diff
@@ -8,19 +9,25 @@ const Header: React.FC = () => {
...
+        <Link to='/' className={classes['logo-link']}>
+          <Logo imgWidth={55} />
+        </Link>
         <Box className={classes.actions}>
-          <IconButton size='large' color='inherit'>
-            <Badge badgeContent={4} color='error'>
-              <Favorite />
-            </Badge>
-          </IconButton>
+          <Link to='/wish-list' className={classes['nav-link']}>
```
Comment (ValeriyaZavozova) [inline src/components/Layout/Header/Header.tsx]: Create a separate enum for your routes and use them as constants in your components

```suggestion
          <Link to={Routes.WISH_LIST} className={classes['nav-link']}>
```

**Review by ValeriyaZavozova — APPROVED** [review]

File: src/components/Layout/Header/Header.tsx (hunk 1 above)
Comment (valkandyba) [reply src/components/Layout/Header/Header.tsx]: fixed

File: src/components/Layout/Header/Header.tsx [hunk 2]
```diff
@@ -1,26 +1,39 @@
...
 import { AppBar, Badge, Box, Button, IconButton, Toolbar } from '@mui/material';
 import { AccountCircle, Favorite, ShoppingBasket } from '@mui/icons-material';
 import Logo from '../../UI/Logo/Logo';
 import classes from './Header.module.scss';
 
+export enum AppRoutes {
+  HOME_PAGE = '/',
+  WISH_LIST = '/wish-list',
+  ORDER = '/order',
+}
```
Comment (ValeriyaZavozova) [inline src/components/Layout/Header/Header.tsx]:
```suggestion
```

Comment (valkandyba) [reply src/components/Layout/Header/Header.tsx]: fixed

**Pull request merged**